l10n_es_facturae/README.rst

```diff
@@ -0,0 +1,108 @@
+desde el día 15 de enero de 2015 todas las facturas remitidas a las
+administraciones públicas tienen que ser electrónicas.
+Téngase en cuenta, no obstante, que muchas Administraciones Públicas,
+y entre ellas la Administración General del Estado, han hecho uso de la
+potestad de exonerar de esta obligación a las facturas de hasta 5.000 euros.
+Estas facturas electrónicas se enviarán a través de los puntos generales de
+entrada de facturas electrónicas de la administración correspondiente.
+Estos puntos generales le permitirán consultar electrónicamente el estado de
+tramitación de su factura, una vez remitida. El de la Administración General
+del Estado se denomina FACe (www.face.gob.es).
+En estas facturas electrónicas habrá que indicar la oficina contable,
+el órgano gestor y la unidad tramitadora, para que llegue correctamente
+a su destino. La administración le proporcionará estos datos.
+
+Informacion sobre el formato:
+
+* http://www.facturae.gob.es/formato/Documents/EspanolFacturae3_0.pdf
+* http://www.facturae.gob.es/formato/Versiones/Esquema_castellano_v3_2_x_17_11_2015_unificado.pdf
+
+Instalación
+===========
+
+Este módulo depende del módulo *account_payment_partner* y sus dependencias,
+que se encuentran en https://github.com/OCA/bank-payment.
+
+Para generar el archivo XML, hace falta el módulo *report_xml* que se encuentra
+en https://github.com/OCA/reporting-engine.
+
+En el caso de querer firmar el formato FacturaE desde Odoo, debe instalarse la
+última versión de xmlsec1, disponible en el repositorio
+https://github.com/lsh123/xmlsec, que debe compilarse e instalarse con todas
+sus dependencias.
+Para que funcione, debe añadirse el parámetro LD_LIBRARY_PATH con el valor
+/usr/local/lib.
+Posteriormente deberá instalarse la libreria xmlsec de Python.
+
+Configuración
+=============
+
+* Es necesario ir a los modos de pago e indicar su correspondencia con los
+  códigos de FACe.
+* La dirección a la que se remite la factura de venta que queremos exportar
+  debe estar marcada como facturae y debe tener cubiertos los datos de
+  Oficina contable, Órgano gestor y Unidad tramitadora.
+* Si se desea firmar el xml generado desde Odoo, tenemos que irnos al
+  formulario de las compañías y subir el certificado en formato .pfx y
+  escribir la contraseña de acceso al certificado.
+
+Uso
+===
+
+Desde el botón "Más" del formulario de factura, ejecutamos el asistente
+denominado "Crear fichero Factura-E"
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/189/8.0
+
+Problemas conocidos / Hoja de ruta
+==================================
+
+* No está soportada la exportación de facturas rectificativas.
+  Fallan las series.
+* El certificado y la contraseña de acceso al certificado no se guardan
+  cifrados en la base de datos.
+* Ver la posibilidad de exportar varias facturas juntas.
+* Soportar formato Factura-E v3.2.1.
```

Comment:
Actualmente son válidos ambos formatos. Si quisieramos habilitar el 3.2.1 deberíamos modificar el report. Como yo tenía experiencia en el 3.2 no quise explorar el 3.2.1, aunque los cambios entre las dos versiones son mínimos. Si lo consideráis necesario se puede hacer la revisión.